plugins/nf-google/src/main/nextflow/cloud/google/batch/GoogleBatchTaskHandler.groovy

```diff
@@ -91,6 +95,11 @@ class GoogleBatchTaskHandler extends TaskHandler implements FusionAwareTask {
      */
     private String uid
 
+    /**
+     * Flag to indicate if task belong to an TaskRunArray
+     */
+    private boolean belongsToArray
```

Comment:
Why not using `Task.isArray` ? 

https://github.com/nextflow-io/nextflow/blob/2a412671df356f801e0ad9a3808e6b99649a362c/modules/nextflow/src/main/groovy/nextflow/processor/TaskRun.groovy#L759-L761

Reply:
If I am not wrong, it indicates if task is a TaskArrayRun but not if the task belongs to an array

Reply:
Only the task array handler is submitted, so it is safe to use `isArray()` during the status check

Reply:
So, this will change the meaning of isArray. Now, it will mean the task is TaskArrayRun or a TaskRun in an array. If you agree, I will move the boolean flag to TaskRun set it false by default and change when it is collected to a TaskArray.

Reply:
yeah, you are right, let's call it `isChild`

Reply:
Ok, I have just renamed it